src/blockdom/block_compiler.ts

```diff
@@ -447,6 +447,33 @@ function buildBlock(template: HTMLElement, ctx: BlockCtx): BlockType {
       }
     };
     B.prototype.beforeRemove = VMulti.prototype.beforeRemove;
+    if (deepRemove) {
+      B = class extends B {
+        children: (VNode | undefined)[] | undefined;
+        beforeRemove() {
```

Comment:
are you sure you need this? would be cool if l449 was enough

Reply:
Indeed, this is not needed, I'm going to remove it.